src/core/ucc_team.c

```diff
@@ -60,7 +61,7 @@ static ucc_status_t ucc_team_create_post_single(ucc_context_t *context,
     team->bp.team                 = team;
     team->bp.map.type             = UCC_EP_MAP_FULL;
     team->bp.map.ep_num           = team->size;
-    team->state                   = UCC_TEAM_ADDR_EXCHANGE;
+    team->state                   = (team->size > 1) ? UCC_TEAM_ADDR_EXCHANGE : UCC_TEAM_ALLOC_ID;
```

Comment:
why not setting team->state to UCC_TEAM_CL_CREATE ? we don't need team id for tl/self, dont we?